sdk/go/common/resource/config/plaintext.go

```diff
@@ -117,6 +152,8 @@ func (c Plaintext) Encrypt(ctx context.Context, encrypter Encrypter) (Value, err
 // encrypt converts the receiver to an object. All secure strings in the result are encrypted using encrypter.
 func (c Plaintext) encrypt(ctx context.Context, path resource.PropertyPath, encrypter Encrypter) (object, error) {
 	switch v := c.Value().(type) {
+	case nil:
+		return object{}, nil
```

Comment:
This seems unrelated to the cleanup.  Is this fixing anything in particular, or just defensive because we've seen similar problems in other places?

Reply:
I just saw https://github.com/pulumi/pulumi/issues/14570, which this will fix :sweat_smile: